packages/arb-token-bridge-ui/src/components/TransferPanel/DepositCardL2Failure.tsx

```diff
@@ -39,17 +85,17 @@ export function DepositCardL2Failure({ tx }: { tx: MergedTransaction }) {
           onClick={() => redeem(tx)}
           className="text-2xl"
         >
-          Re-execute
+          Retry for free
```

Comment:
This is wrong, as it's not free (only the auto-redeem done automatically by the system is free) - let's jut keep the old text.

Reply:
Done. Updated according to the new design.